vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+
+We can fit (1) by least squares and test $H_0: \beta_1 = 0$ to obtain a marginal test for each category $j$; in this case, each chromosome band. The procedure also generalizes to incorporate the nesting structure of chromosome bands. Specifically, if band $j_2$ (e.g., 12q21.1) is nested within a coarser band $j_1$ (e.g., 12q21) and the more granular band $j_2$ is significant, then the effect of membership in $j_1$ over and above the effect attributable to membership in $j_2$ can be tested by fitting the model $$\label{eq:lm2}
+  y_i = \beta_0 + \beta_1 a_{ij_1} + \beta_2 a_{ij_2} + \varepsilon_{i}$$ and testing the null hypothesis $H_0: \beta_1 = 0$. Multiple significant sub-bands and multiple levels of nesting can be incorporated by including further terms in the model. The complete process can be summarized as follows: Start by marginally testing each band which has no sub-bands. For all other bands, first test all sub-bands, then test the current band using a linear model that includes a term for each *significant* sub-band.
+
+We apply this procedure to perform global tests using per-gene $t$-statistics as a measure of differential expression in BCR/ABL relative to NEG samples. As with the Hypergeometric tests, we start by creating objects of class *"ChrMapLinearMParams"*.
+
+```{r LMtestSetup}
+
+params <- new("ChrMapLinearMParams",
+              conditional = FALSE,
+              testDirection = "up",
+              universeGeneIds = entrezUniverse,
+              geneStats = tstats,
+              annotation = "hgu95av2",
+              pvalueCutoff = 0.01, 
+              minSize = 4L)
+
+params@graph <- makeChrBandGraph(params@annotation, params@universeGeneIds)
+params@gsc <- makeChrBandGSC(params@graph)
+paramsCond <- params
+paramsCond@conditional <- TRUE
+```
+
+The tests are performed, and the results summarized by
+
+```{r LMtest}
+lmans <- linearMTest(params)
+lmansCond <- linearMTest(paramsCond)
+chrSortOrder(summary(lmans))
+
+chrSortOrder(summary(lmansCond))
+##
+```
+
+<figure id="fig:dotplot1p362">
+
+```{r echo=FALSE}
+tmpSet <- subsetByBand(nsFiltered, ct, "1p36.2")
+esetBWPlot(tmpSet, ylab="1p36.2", layout = c(2, 8),
+           par.strip.text = list(cex = 0.8))
+```
+
+<figcaption>Figure 3: Expression values for the BCR/ABL and NEG samples for the genes located within 1p36.2, which is declared as significant by the Hypergeometric test, but not by GSEA. Genes in the selected list are highlighted with a shaded background. For these genes, The NEG samples, those with no known cytogenetic abnormalities, have significantly lower expression than the BCR/ABL samples. However, the direction is reversed (albeit mildly) for many of the other genes.</figcaption>
+</figure>
```

Comment:
Lines 356-365 should be modeled after Figures 1 and 2.

Reply:
**Please check the pdf, I just converted from swerve and edit as the pdf like you  instructed us, I have no control of the codes and I dot understand what you are asking me to do, Kindly check the Pdf and the Sweave codes.**
 **I dont know those codes or what they do, I just did as told, covert swerve, edit to look like pdf and every edit you asked I have done so I don't know. So please clarify and kindly make up your mind .**